psi4/driver/procrouting/proc.py

```diff
@@ -1499,21 +1509,27 @@ def scf_helper(name, post_scf=True, **kwargs):
     # The wfn from_file routine adds the npy suffix if needed, but we add it here so that
     # we can use os.path.isfile to query whether the file exists before attempting to read
     read_filename = scf_wfn.get_scratch_filename(180) + '.npy'
+    read_mos = ((core.get_option('SCF', 'GUESS') == 'READ') and os.path.isfile(read_filename)) or ref_wfn is not None
```

Comment:
If we have set `set guess read` and `read_filename` is not a file, we get a SAD guess without warning right now, right? Maybe we should print a warning.

Reply:
That should be catched by the `elif` below that prints a warning.